repondeur/zam_repondeur/tasks/fetch.py

```diff
@@ -38,24 +38,32 @@ def fetch_articles(lecture_pk: int) -> None:
 
 @huey.task(retries=3, retry_delay=RETRY_DELAY)
 def fetch_amendements(lecture_pk: int) -> None:
-    with transaction.manager, huey.lock_task(f"fetch-{lecture_pk}"):
```

Comment:
The thing we’re trying to make short here is the database transaction, not the Huey lock

Reply:
🤐